lib/toolkit/manifest.rb

```diff
@@ -66,6 +66,12 @@ def shell?(name)
       File.basename(ENV['SHELL']) == name
     end
 
+    # Tests whether the OS matches the argument.
+    def os?(osName)
+      os = `uname -s`.strip
+      os.eql?osName
```

Comment:
I'm curious why you didn't use `os == osName` here.

Reply:
I think it is just that this is the second thing I have ever written in Ruby.  I will clean it up.

I also had to do some interesting simlink games to make this work with my use cases which was firefox settings needing to land in different folders depending on the OS, maybe there is a better way?

```
drwxr-xr-x  4 gorzell  staff   128B Nov 27 12:44 firefox
lrwxr-xr-x  1 gorzell  staff     7B Nov 28 12:40 firefox-mac -> firefox
lrwxr-xr-x  1 gorzell  staff     7B Nov 28 12:40 firefox-nix -> firefox
```
```
# Firefox
package 'firefox-mac', :when => os?('Darwin'), :into => 'Library/Application Support/Firefox/Profiles/default'
package 'firefox-nix', :when => os?('Linux'), :into => '.mozilla/firefox/default'
```

Reply:
> I think it is just that this is the second thing I have ever written in Ruby.

Hah, fair enough. Even more props for contributing, then. 😀 

> firefox settings needing to land in different folders depending on the OS

Hmm - if the files are meant to be the same, try this; it's not pretty, but captures the logic:

```ruby
package 'firefox',
  :when => os?('Darwin') || os?('Linux'),
  :into => os?('Darwin') ? 'Library/Application Support/Firefox/Profiles/default' : '.mozilla/firefox/default'
```
Or, if you're always on one of the two OSes you could drop the `:when` condition and turn it on by default.